projects/2021_05_09_user_operated_internet_fund.md

```diff
@@ -508,6 +508,187 @@ Metrics regarding applications to the October call
+      
+
+#### RADIUSdesk
+
+* Project URL: https://NLnet.nl/project/Radiusdesk
+* Current status: Preparation
+* Deliverables to date: -
+
+##### Executive summary
+
+  RADIUSdesk and MESHdesk help to set up and manage mesh networks at scale, and are open source from top to bottom. They can be used in tandem to provide public wifi, or set up mesh networks as well as community networks. Allowing someone to flash a cheap access point and then managing it irrespective of the hardware vendor offers great opportunity for poorer communities to enable themselves in terms of providing Internet access. Existing hardware that reached end-of-life can be managed in a similar way (often much simpler) than what the vendors offer. Because there is a RADIUS server included, there is a single integrated system which is able to manage connections as well as the hardware. This enables anyone to set up an end-to-end system that can provide Internet access, with OpenStreetmap integration, alerts, and other advanced features.
+
+The following projects from M1 and M2 continued:
+
+#### Armbian
+(See M1 report, https://nlnet.nl/project/Armbian)
+
+#### CeroWRT II
+(See M1 report, https://nlnet.nl/project/CeroWRT-II)
+
+#### Open source ePDG for VoWiFi
+(see M1 report, https://nlnet.nl/project/Osmocom-ePDG)
+
+#### Telecommunication in HF using the Internet Protocol (IPoHF)
+* See M2 report: https://nlnet.nl/project/IPoHF
+
+#### WebRTC network congestion management for PION
+* See M2 report: https://nlnet.nl/project/Pion-adaptive
+* Software delivery: https://github.com/mengelbart/syncodec
+
+
+An up-to-date overview of projects supported by UOF can always be found on:
+
+https://nlnet.nl/thema/User-operatedInternetFund.html
+
+### 7. Support projects
+
+For each active funded project which is classified as a support project: a management overview of the use of resources, remaining budget, and which projects were supported.
+
+* **Radically Open Security - Security scans, audits and advice**
+  - Use of resources:
+      * 0€ claimed
+      * Remaining budget: 10 000€
+
+### 8. Attestation
+
+I attest on my honor that the above transaction information is true and accurate and that the payments to projects have been made after the agreed results have been delivered
+  * **signature on PDF version of this document**
+ 
+### 9. Metrics
+
+Metrics regarding applications to the 2022-02U call:
+* Number of applications: **12**
+* Total budget requested: **480,389**€
```

Comment:
Place the € at the front of amount.

Reply:
In Europe, the € is placed after the number. But anyway these types of issues are not what we're looking for, more like anything which could be used to argue that the payment is unjustified.